source/agora/test/Script.d

```diff
@@ -198,8 +201,8 @@ unittest
         WK.Keys.Genesis.address.repeat.take(8)).sign()).array;
 
     /// using two different key pairs for the IF / ELSE branch
-    auto kp_a = WK.Keys[42];
-    auto kp_b = WK.Keys[69];
+    const KeyPair kp_a = WK.Keys[42];
+    const KeyPair kp_b = WK.Keys[69];
```

Comment:
Why the diff ?

Reply:
It needed to be available at compile time for the template.